proto/recon.proto

```diff
@@ -145,3 +145,15 @@ message RecognizePlacesResponse {
   }
   repeated Item items = 1;
 }
+
+// RecogPlace contains info of a place for RecognizePlaces method.
+message RecogPlace {
+  repeated string words = 1;
+  string dcid = 2;
+  repeated string containing_places = 3;
+  int64 population = 4;
+}
```

Comment:
Just wonder why this is proto and not struct?  Also, is recon.proto an internal proto

Reply:
This is to make unit test easier. Deep comparison of proto messages is cleaner than deep comparison of structs.
recon.proto could be public I think? I don't see a reason why we want to keep it private.